setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+  name = 'bplist',
+  packages = ['bplist'],  
+  version = '0.1',
```

Comment:
You can safely tag this as 1.0, it didn't change in years and is extremely stable (as in, no changes are coning up)